### ISSUE TITLE
[Bug Report][3.7.2] V-autocomplete vertical direction

### ISSUE THREAD

**Issue by Doltario** [vuetifyjs/vuetify#20604]
### Environment
**Vuetify Version:** 3.7.2
**Vue Version:** 3.4.26
**Browsers:** Firefox 131.0
**OS:** Mac OS 10.15

### Steps to reproduce
Open the link, notice that v-autocomplete width seems really small. It occurs when the `direction="vertical"` prop is added.  

### Expected Behavior
Width to be usable.

### Actual Behavior
The width is unsetable and really small, so that user can't even see the selected values.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrVUj1PwzAQ/SuWlyyFwMAShUpVJfgDSAxNB+Mc7UkXn2VfQivEf8dxqeqOjF2sd++e70tv861jsPXK+/tpBN3oVmDwZASWnVOqne6M9xnmwLITgw7CH3VSjMKW0y8QONNK2T36eAl7DGAF2T13eoIgaA11+pIn8wGUcquyWJFvMA0Wk2BTrQ3hJweHplqoas3EwfQ845cEsc/0K3DYnRRvcDBxBu9HHtDtqm1ZeBhJMLU7M8u2vl7p36tGORKkSb+wl32jHp8e/KHsePOnmOlrJ2RhNkpbFwbSPws3EuVn+wtXEMAK)

**Comment by yogender-kulshrestha** [comment]
If your goal is to display the selected chips in the v-autocomplete vertically, instead of using direction="vertical", you can apply a custom CSS rule to achieve the same layout.

You can add the following custom CSS to handle the vertical stacking of chips:

```
.custom-width .v-field__input {
  display: grid;
}
```

This will ensure that the selected chips are displayed in a vertical stack, without causing the width issue that occurs when using the direction="vertical" prop. This approach allows you to maintain control over the width and layout while achieving the desired vertical chip display.

Solution Link

[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrNUsFqwzAM/RWTSy5tcs+yQilsPzDYoS3Fs91UoNjGlrOF0n+f46zEhV1228VIz096erb218I7UW+trYagiqZoSfUWOanNQTPWDmtubQpTIowmDlq5H2hmBDLCxCpF6g4zJi5g/ZIi/1D4fCi2OblY7huIwj4S9uWOI5yN08DLFSt3Bo3j0kzxSwxBJvhVGdfNjDf1xf0UvI+mB92Vx7xxH5AgymWTIfeTkgieTL/+BEmXpWLT1o+WotW/mpXglCAwOqoMyhEIjvlM/+YxfjObjEb48a8TMa1CW2crElNPI851Vf6irBrWZ1AoTyfQNhC7zq0l+Fg8NqyL0z9N2G3qee9S3FY6IKbj+A2r8tQO)

**Comment by Doltario** [comment]
Hi, thanks for your answer and for providing a demo link.

Your workaround may work, nevertheless it is still a bug that should be fixed. The prop as it is, is not usable at all.

**Comment by MatthewAry** [comment]
Hi there, I tried to track down what this prop is supposed to do from a logical standpoint and it seems to be inherited from the VInput component. Please see:

- https://github.com/vuetifyjs/vuetify/blob/ca85fc0bd098a27bcd57aa7acf51e85968061a74/packages/vuetify/src/components/VInput/VInput.tsx#L56-L60
- https://github.com/vuetifyjs/vuetify/blob/ca85fc0bd098a27bcd57aa7acf51e85968061a74/packages/vuetify/src/components/VInput/VInput.tsx#L163
- https://github.com/vuetifyjs/vuetify/blob/ca85fc0bd098a27bcd57aa7acf51e85968061a74/packages/vuetify/src/components/VInput/VInput.sass#L22-L31

After looking at it, (I'm not super experienced with CSS Grids) my best guess is that this prop is meant to vertically stack the Append, Control, and Prepend slots on the component, not vertically stack the contents of the control. Clearly using it can break components. My take? This prop should not be inherited by VSelect, VCombobox, or VAutocomplete, and probably others. It's a low level prop in VInput.

I will mark this as a bug because a correction is required, even if it's not going to be the fix you're looking for.

TLDR: This prop is probably not meant to do what you're hoping it will do.

**Comment by J-Sek** [comment]
Resolved by #22519, the prop was just leaking by accident - not a feature we meant to implement in regular text fields.
